department-of-veterans-affairs/abd-vro
# [abd-vro #2143] Update Github Actions with Java 17 Install

## What was the problem?
During the upgrade to springboot 3 we also indrectly required new versions of Java to use the updated gradle dependencies. This caused some build steps to fail and required manual intervention to the CI/CD process.

Associated tickets or Slack threads:
https://github.com/department-of-veterans-affairs/abd-vro/issues/2143

## How does this fix it?[^1]

The PR adds the steps to install Java 17, which fixed the issue for other actions (test-code workflow)
## How to test this PR
- Run lint-gradle-files workflow and create-release-tag
- If the steps finish without issue the changes worked.


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
